db/structure.sql

```diff
@@ -106,3 +106,21 @@ CREATE TABLE `values` (
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 
+--
+-- Table structure for table `resources`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `resources` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `slug` varchar(255) NOT NULL,
+  `type` varchar(255) NOT NULL,
+  `value_string` varchar(255) DEFAULT NULL,
+  `value_long_string` text,
+  `value_double` decimal(19,4) DEFAULT NULL,
+  PRIMARY KEY (`id`)
```

Comment:
Put an index on `key` as well, since we will be using it often e.g. `api/resources/some_key`

```suggestion
  PRIMARY KEY (`id`),
  INDEX `idx_key` (`key`)
```